src/credentials/FabricTable.cpp

```diff
@@ -641,9 +646,15 @@ void FabricTable::DeleteAllFabrics()
     }
 }
 
-CHIP_ERROR FabricTable::Init(FabricStorage * storage)
+CHIP_ERROR FabricTable::Init(FabricStorage * storage, FabricListener * fabricListener)
 {
     VerifyOrReturnError(storage != nullptr, CHIP_ERROR_INVALID_ARGUMENT);
+    if (fabricListener != nullptr)
+    {
+        mFabricListener   = std::move(fabricListener);
```

Comment:
This std::move is meaningless, for a pointer type....

Reply:
removed